### PR TITLE
apply sorting on multiple applications table employer

### PR DESCRIPTION
**Pull request by DavidRimarDfE**

### CODE REVIEW

File: src/Employer/UnitTests/Employer.Web/Controllers/ApplicationReviewsControllerTests.cs
Comment (dashton82) [inline src/Employer/UnitTests/Employer.Web/Controllers/ApplicationReviewsControllerTests.cs:1]: Would be good if we could also get a test in to cover the flow of what happens if the sort column and sort order arent valid enum values

File: src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs [hunk 1]
```diff
@@ -26,20 +27,26 @@ public ApplicationReviewsController(IApplicationReviewsOrchestrator orchestrator
...
+
+            var viewModel = await _orchestrator.GetApplicationReviewsToUnsuccessfulViewModelAsync(rm, outputSortColumn, outputSortOrder);
 
             return View(viewModel);
         }
 
         [FeatureGate(FeatureNames.MultipleApplicationsManagement)]
         [HttpPost("unsuccessful", Name = RouteNames.ApplicationReviewsToUnsuccessful_Post)]
-        public async Task<IActionResult> ApplicationReviewsToUnsuccessfulAsync(ApplicationReviewsToUnsuccessfulRouteModel rm)
+        public async Task<IActionResult> ApplicationReviewsToUnsuccessfulAsync(ApplicationReviewsToUnsuccessfulRouteModel rm, [FromQuery] string sortColumn, [FromQuery] string sortOrder)
```
Comment (dashton82) [inline src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs:42]: is there a reason why this isnt on the model being posted? for the sortColumn and sortOrder?

Comment (DavidRimarDfE) [reply src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs:42]: The reason is that if the user does not select any checkboxes, then the server-side sorting is "preserved", it basically goes to the db again to get the same applications sorted in the same order. Not sure if there is a way around it? Open to suggestions. Maybe the tech debt ticket EC-114 would be a solution, removing the modelstate validations?

File: src/Employer/UnitTests/Employer.Web/Controllers/ApplicationReviewsControllerTests.cs
Comment (DavidRimarDfE) [reply src/Employer/UnitTests/Employer.Web/Controllers/ApplicationReviewsControllerTests.cs:1]: added a test case in my last commit, it'll basically default to the Default enum value, amended the setup to match what it's gonna get

File: src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs (hunk 1 above)
Comment (DavidRimarDfE) [reply src/Employer/Employer.Web/Controllers/ApplicationReviewsController.cs]: No reason. I moved it inside the posted model

**Review by dashton82 — APPROVED** [review]

**Pull request merged**